### ISSUE TITLE
very low performance

### ISSUE THREAD

**Issue by minhthai1995** [Tencent/NeuralNLP-NeuralClassifier#106]
I tried to use the glove.6B.300d.txt for the pre-trained embedding but the performance of the model on the RCV-1 dataset was poor. Could you share with me the correct conf?
Best test performance at epoch 1 is precision: 0.756123, recall: 0.411358, fscore: 0.532835, macro-fscore: 0.236519, right: 5063, predict: 6696, standard: 12308.
Loss is: 0.052504.

**Comment by MagiaSN** [comment]
Hello, we obtained our best results on RCV-1 using TextRCNN with two-layers Bi-GRU and one-layer CNN, Adam optimizer and learning of 8e-3, as described in the paper. You should probably train for more epochs on multi-label classification dataset like RCV-1.
